.gitattributes

```diff
@@ -0,0 +1,7 @@
+# mocked files
+mock_*.go binary
+*_mock.go binary
```

Comment:
Just as a query: what's the reasoning behind hinting to Go that these generated source files are binary?